src/tasks/helpers/getContainer.js

```diff
@@ -96,23 +98,25 @@ var augmentModule = function(modulesOutput, extensionName, extensionPath, module
   });
 };
 
-var augmentModules = function(extensionOutput, extensionDescriptor, extensionPath) {
+const augmentModules = function(extensionOutput, extensionDescriptor, extensionPath) {
   extensionOutput.modules = extensionOutput.modules || {};
 
   FEATURE_TYPES.forEach(function(featureType) {
     if (extensionDescriptor.hasOwnProperty(featureType)) {
-      var featureDescriptors = extensionDescriptor[featureType];
+      const featureDescriptors = flatten([extensionDescriptor[featureType]]);
```

Comment:
If you did this you could probably simplify the rest of the code (you wouldn't have to do `featureDescriptor.libPath || featureDescriptor);`) :

```
let featureDescriptors = extensionDescriptor[featureType];

if (typeof featureDescriptors === 'string') {
  featureDescriptors = [{
    libPath: featureDescriptors
  }]
}
```